mtools/mlaunch/mlaunch.py

```diff
@@ -1422,7 +1440,11 @@ def _construct_mongos(self, logpath, port, configdb):
             extra = self._filter_valid_arguments(self.unknown_args, "mongos") + extra
 
         path = self.args['binarypath'] or ''
-        command_str = "%s --logpath %s --port %i --configdb %s --logappend %s %s --fork"%(os.path.join(path, 'mongos'), logpath, port, configdb, auth_param, extra)
+	if os.name == 'nt':
+            newLogPath=logpath.replace('\\', '\\\\')
+            command_str = "start /b %s --logpath %s --port %i --configdb %s --logappend %s %s "%(os.path.join(path, 'mongos'), logpath, port, configdb, auth_param, extra)
+	else:
+            command_str = "%s --logpath %s --port %i --configdb %s --logappend %s %s --fork"%(os.path.join(path, 'mongos'), newLogPath, port, configdb, auth_param, extra)
```

Comment:
I believe `newLogPath` is not defined when the OS is not Windows, resulting in the error `UnboundLocalError: local variable 'newLogPath' referenced before assignment`